projects/addon-editor/components/color-selector/palette/palette.template.html

```diff
@@ -1,5 +1,6 @@
+<!--TODO remove any in v3.0 (after update angular to 10.1.0+)-->
 <button
-    *ngFor="let color of colors | keyvalue"
+    *ngFor="let color of $any(colors) | keyvalue"
```

Comment:
https://github.com/angular/angular/commit/3373453

Reply:
add TODO please in which version of angular to remove this

Reply:
already. 2 lines above